Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -593,6 +593,22 @@ else()
   install(TARGETS dolphin-emu RUNTIME DESTINATION ${bindir})
 endif()
 
+if (CMAKE_SYSTEM_NAME STREQUAL "Linux" AND STEAM)
+  # Set that we want ORIGIN in FLAGS.
```

Comment:
Could possibly use `BUILD_RPATH_USE_ORIGIN` though that wouldn't allow you to disable the tags...

Reply:
I think I tried that earlier and it used RUNPATH?